network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/config/WgVpnRoutes.kt

```diff
@@ -62,7 +115,11 @@ internal class WgVpnRoutes {
             "169.255.0.0" to 16,
             "170.0.0.0" to 7,
             "172.0.0.0" to 12,
-            // Excluded range: 172.16.0.0 -> 172.31.255.255
+            // Excluded range: 172.16.0.0 -> 172.16.255.255
```

Comment:
Why are we excluding this range too?

Reply:
This was a mistake, the right range was what it was before, ie. 172.16.0.0/12. I probably mistyped `16` instead of `12` when generating the routes